src/index.js

```diff
@@ -2,6 +2,14 @@ import { parse } from 'acorn-loose';
 import { readFileSync } from 'fs';
 import { simple, full, ancestor } from 'acorn-walk';
 
+const returnSuggestions = () => {
```

Comment:
export returnSuggestions and getCursorPosition in line with replacing require and using imports